python-checks/src/test/resources/checks/cdk/s3BucketVersioning.py

```diff
@@ -0,0 +1,42 @@
+from aws_cdk import aws_s3 as s3
+
+a = int(10)
```

Comment:
I would put this call to the end of the file and choose a better function name like `function_call_for_coverage` to show the intention of this call